schema/common/defaults/configuration/node-exporter.yml

```
@@ -32,10 +32,10 @@ specification:
     - "--web.listen-address=:9100"
     - '--log.level=info'
     - '--collector.diskstats.ignored-devices=^(ram|loop|fd)\d+$'
-    - '--collector.filesystem.ignored-mount-points=^/(sys|proc|dev|run)($|/)'
-    - '--collector.netdev.device-blacklist="^$"'
+    - '--collector.filesystem.mount-points-exclude=^/(sys|proc|dev|run)($|/)'
+    - '--collector.netdev.device-exclude="^$"'
     - '--collector.textfile.directory="/var/lib/prometheus/node-exporter"'
```

this should probably be
```suggestion
    - '--collector.textfile.directory=/var/lib/prometheus/node-exporter'
```

---

https://github.com/epiphany-platform/epiphany/issues/1681#:~:text=Error%20reading%20textfile%20collector%20directory%20%5C%22/var/lib/prometheus/node%2Dexporter%5C%22%3A